### ISSUE TITLE
prescript issue

### ISSUE THREAD

**Issue by pkra** [krautzource/sre-to-tree#36]
Something like `\prescript {h_{l-1}}{}{P}` ends up producing invalid output (role presentation, no label but aria-level/posinset etc).

Let's track this here, though this might turn out to be an SRE problem.

**Comment by pkra** [comment]
I've filed https://github.com/Speech-Rule-Engine/speech-rule-engine/issues/726
